src/configuration.cc

```diff
@@ -33,6 +33,7 @@ constexpr const int kMetadataApiDefaultNumThreads = 3;
 constexpr const int kMetadataApiDefaultPort = 8000;
 constexpr const char kMetadataApiDefaultResourceTypeSeparator[] = ".";
 constexpr const int kMetadataReporterDefaultIntervalSeconds = 60;
+constexpr const int kMetadataReporterDefaultPurgeDeleted = false;
```

Comment:
Should we be defaulting to true?

Reply:
I'd rather put this through a few more tests. It can be overridden in the config map if necessary.